projects/observability/src/pages/api-trace-detail/api-trace-detail.page.component.ts

```diff
@@ -31,12 +33,19 @@ import { ApiTraceDetails, ApiTraceDetailService } from './api-trace-detail.servi
             icon="${IconType.Time}"
             [value]="traceDetails.timeString"
           ></ht-summary-value>
-          <ht-summary-value
-            class="summary-value"
-            icon="${IconType.TraceId}"
-            label="Trace ID"
-            [value]="traceDetails.traceId"
-          ></ht-summary-value>
+
+          <ng-container
+            *ngIf="this.getExploreNavigationParams(traceDetails.traceId) | async as exploreNavigationParams"
```

Comment:
1 nit, 1 major here:
nit - the ngIf can move down onto the link, the extra container is unnecessary. 
major - calling a function to return an observable will return a new observable each time - so every cycle will be treated as a change from the async pipe's perspective. Instead, you can either assign the observable to a variable and reference that here, or use `htMemoize` pipe before the async one. My guess is the former will be clearer rather than chaining pipes.

Reply:
Both this comment and the one about the link component came up in the PR to add this support on span attributes, so you could use that PR as a reference too (although there, the decision was made for styling reasons to keep the filter link and value rendered separately - not sure if the same applies here):

https://github.com/hypertrace/hypertrace-ui/pull/1079/files#diff-8912ac915c8ff140908e47b3997b34e34b6eb5a0defd418b07953b86980ba4c5R19-R24